core/src/main/java/org/apache/druid/java/util/common/StringUtils.java

```diff
@@ -40,6 +40,7 @@
 public class StringUtils
 {
   public static final byte[] EMPTY_BYTES = new byte[0];
+  public static final String LINE_SEPARATOR = System.lineSeparator();
```

Comment:
Since there is a direct method in `System` (rather than getProperty(...)) there is no point in storing it in a constant. Please use directly.